_dev/src/components/product-feed/settings/commons/actions-buttons.vue

```diff
@@ -49,11 +49,16 @@ export default {
     nextStep: {
       type: Function,
       required: true,
+      default() {
+        return 1;
+      },
     },
     previousStep: {
       type: Function,
       required: false,
-      default: null,
+      default() {
+        return 1;
+      },
```

Comment:
```suggestion
      default: () => {},
```

This way, we know there is nothing done or returned